src/controllers/AuthController.js

```diff
@@ -0,0 +1,94 @@
+import { hashSync, genSaltSync } from 'bcryptjs';
+import Mailer from '../utils/mailer';
+import db from '../models';
+import Helpers from '../utils/helpers';
+import { ApiResponse } from '../utils/index';
+
+
+const { generateToken, verifyToken } = Helpers;
+const { User } = db;
+const { sendResetMail } = Mailer;
+
+/**
+ * Collection of methods for Auth
+ * @class AuthController
+ */
+class Auth {
+  /**
+   * Sends a user reset password link
+   *
+   * @param {Request} req - The request from the endpoint.
+   * @param {Response} res - The response returned by the method.
+   * @returns {JSON} A JSON response with a successfully message.
+   * @memberof Auth
+   */
+  static async sendResetPasswordEmail(req, res) {
+    try {
+      const { email } = req.body;
+      const user = await User.findOne({ where: { email } });
```

Comment:
Direct database query should not be made in the controllers, should use a service method here